python-avd/pyavd/_schema/avddataconverter.py

```diff
@@ -191,17 +193,17 @@ def convert_types(self, convert_types: list, data: dict | list, index: str | int
 
                     yield AvdConversionWarning(key=path, oldtype=convert_type, newtype=schema_type)
 
-    def deprecation(self, deprecation: dict, _, __, path: list):
+    def deprecation(self, deprecation: dict, _data: Any, _schema: dict, path: list) -> Generator:
         """
-        deprecation:
+        deprecation.
+
           warning: bool, default = True
           new_key: str
           removed: bool
           remove_in_version: str
           remove_after_date: str
-          url: str
+          url: str.
```

Comment:
hmmm - would that work?
```suggestion
          url: str
          
```